editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -596,6 +598,42 @@ export const previewStore = (
   path: ApplicationPath.SingleSession,
 
   saveToEmail: undefined,
+
+  overrideAnswer: (fn: string) => {
+    // Similar to 'changeAnswer', but enables navigating backwards to and overriding a previously **auto-answered** question which would typically be hidden
+    const { breadcrumbs, flow, record, changeAnswer } = get();
+
+    // Iterate through breadcrumbs and find the first nodeId that submitted this passport value (eg FindProperty)
+    let originalNodeId;
+    Object.entries(breadcrumbs).forEach(([nodeId, breadcrumb]) => {
+      if (breadcrumb.data && fn in breadcrumb.data) {
+        originalNodeId = nodeId;
+        return;
+      }
+    });
+
+    if (originalNodeId) {
+      // Omit existing fn from breadcrumbs in whichever component originally set it, so it won't be auto-answered in future
+      record(originalNodeId, {
+        data: omit(breadcrumbs?.[originalNodeId]?.data, fn),
+      });
+    }
+
+    // Iterate through breadcrumbs and find the first nodeId that will set this passport value (eg Question "What type of property is it?")
+    //   leave this node's data intact in the breadcrumbs so that the original answer is highlighted later
+    let overrideNodeId;
+    Object.entries(breadcrumbs).forEach(([nodeId, breadcrumb]) => {
+      if (flow[nodeId].data?.fn === fn || flow[nodeId].data?.val === fn) {
+        overrideNodeId = nodeId;
+        return;
+      }
+    });
+
+    if (overrideNodeId) {
+      // Travel backwards to the "override" nodeId to manually re-answer this question, therefore re-setting the passport value onSubmit
+      changeAnswer(overrideNodeId);
+    }
+  },
```

Comment:
felt a little clumsy coming up with clear names here - very open to suggestions for more clear ways to describe "originalNodeId" (this may also have to become plural, still testing repeated "change" clicks to see if there's any edge cases) & "overrideNodeId" !

Reply:
went for inline comments here & added more explicit types!